admin/client/src/styles/_typography.scss

```diff
@@ -58,3 +59,10 @@
 		font-family: 'Bitstream Vera Sans Mono','Courier', monospace;
 	}
 }
+
+.label {
```

Comment:
Its a bootstrap class which is already top level so if it was being used it would affect things already.